seqr/views/apis/anvil_workspace_api.py

```diff
@@ -31,6 +32,12 @@
 
 BLOCK_SIZE = 65536
 
+ANVIL_LOADING_EMAIL_DATE = '2021-12-13'
+ANVIL_LOADING_DELAY_EMAIL = """
+We have received your request to load data to seqr from AnVIL. Currently, the Broad Institute is holding an internal 
+retreat or closed for the winter break so we are unable to load data until the 2nd week of January 2022. We appreciate 
+your understanding and support of our research team taking some well-deserved time off and hope you also have a nice break.
+"""
```

Comment:
It would be ideal to set with environment variables. Then, we can add and remove the warning without updating the software.

Reply:
it is substantially easier to do a code redeploy than an env variable redeploy in seqr. This was designed on purpose to be as easy as possible for. me